sw/device/tests/power_virus_systemtest.c

```diff
@@ -141,9 +142,12 @@ enum {
   kI2c1DeviceAddress1 = 0x44,
   kI2c2DeviceAddress0 = 0x55,
   kI2c2DeviceAddress1 = 0x66,
+  kI2c3DeviceAddress0 = 0x77,
+  kI2c3DeviceAddress1 = 0x88,
```

Comment:
FYI, this isn't a valid address--We only get 7 bits, hehe.

Reply:
Indeed, thanks :) Fixed